appserver/connectors/work-management/src/main/java/com/sun/enterprise/connectors/work/WorkCoordinator.java

```diff
@@ -217,7 +225,7 @@ public void preInvoke() {
                     new WorkEvent(this, WorkEvent.WORK_STARTED, work, null));
         }
 
-        //set the unauthenticated securityContext before executing the work            
```

Comment:
Traling spaces are evil! Another automatic fix. And I have also script which could do it in all java sources, maybe I will execute it as a xmass present :-)